11-deepfloyd-text-image-model/main.py

```diff
@@ -0,0 +1,83 @@
+import base64
+from io import BytesIO
+from diffusers import DiffusionPipeline
+from diffusers.utils import pt_to_pil
+import torch
+
+from pydantic import BaseModel
+
+
+
+########################################################################
+# Insert your parameters here.
+########################################################################
+class Item(BaseModel):
+    prompt : str
+    # image:  Optional[]
+    # url:  Optional[str]
+
+
+########################################################################
+# Run once on first deployment to initialise the model
+########################################################################
+
+access_token = '<<<<Your Access Token Here>>>>'
```

Comment:
@Katsie011 could be more specific. I'm worried users will think its their cerebrium api key